lib/Controller/SignalingController.php

```diff
@@ -150,6 +150,13 @@ public function pullMessages($token) {
 		$seconds = 30;
 		$sessionId = '';
 
+		try {
```

Comment:
What is your ping timeout? If it's around 30 seconds or less, a user might get removed from the room if no messages arrive for the 30 seconds the `pullMessages` waits. The next ping will only happen on the next client request which is 30 seconds + roundtrip time away.

Maybe it's better to `ping` inside the `while ($seconds > 0)` below - every 10 seconds or similar.

Reply:
We changed the timeout to 3*ping+10 (100 seconds)